masonry/src/passes/update.rs

```diff
@@ -492,6 +493,18 @@ fn update_new_widgets(
         // The widget will call `RegisterCtx::register_child` on all its children,
         // which will add the new widgets to the arena.
         widget.item.register_children(&mut ctx);
+
+        #[cfg(debug_assertions)]
+        for child_id in widget.item.children_ids() {
+            if widget.children.get_child(child_id.to_raw()).is_none() {
+                panic!(
+                    "Error in '{}' #{}: method register_children() did not call RegisterCtx::register_child() on child #{} returned by children_ids()",
```

Comment:
Is this string long enough that it might break `cargo fmt`?

Reply:
It passes CI, so presumably not.

Reply:
Nah, `cargo fmt` silently fails on that.

Reply:
What do you mean by silently fails? As in, fail to break the line?

Reply:
Usually it will also fail to format some of the surrounding code too!

You can detect this by adding this to `rustfmt.toml`, but they're not stable, so have to use `cargo +nightly fmt`:

```
error_on_line_overflow = true
error_on_unformatted = true
```